python/paddle/fluid/tests/unittests/dygraph_to_static/test_resnet.py

```diff
@@ -0,0 +1,342 @@
+        y = self.conv(inputs)
+        y = self.pool2d_max(y)
+        for bottleneck_block in self.bottleneck_block_list:
+            y = bottleneck_block(y)
+        y = self.pool2d_avg(y)
+        y = fluid.layers.reshape(y, shape=[-1, self.pool2d_avg_output])
+        pred = self.out(y)
+
+        loss = fluid.layers.cross_entropy(input=pred, label=label)
+        avg_loss_ = fluid.layers.mean(x=loss)
+        acc_top1_ = fluid.layers.accuracy(input=pred, label=label, k=1)
+        acc_top5_ = fluid.layers.accuracy(input=pred, label=label, k=5)
+
+        return pred, avg_loss_, acc_top1_, acc_top5_
+
+
+def train_resnet_in_static_mode():
+    """
+    Tests model decorated by `dygraph_to_static_output` in static mode. For users, the model is defined in dygraph mode and trained in static mode.
+    """
+
+    exe = fluid.Executor(place)
+    startup_prog = fluid.Program()
+    main_prog = fluid.Program()
+
+    with fluid.program_guard(main_prog, startup_prog):
+
+        img = fluid.data(name="img", shape=[None, 3, 224, 224], dtype="float32")
+        label = fluid.data(name="label", shape=[None, 1], dtype="int64")
+        label.stop_gradient = True
+        resnet = ResNet()
+        pred, avg_loss_, acc_top1_, acc_top5_ = resnet(img, label)
+        optimizer = optimizer_setting(parameter_list=resnet.parameters())
+        optimizer.minimize(avg_loss_)
+
+    exe.run(startup_prog)
+
+    train_reader = paddle.batch(
+        paddle.dataset.flowers.train(use_xmap=False), batch_size=batch_size)
+
+    for epoch in range(epoch_num):
+        total_loss = 0.0
+        total_acc1 = 0.0
+        total_acc5 = 0.0
+        total_sample = 0
+
+        for batch_id, data in enumerate(train_reader()):
+            start_time = time.time()
+            dy_x_data = np.array(
+                [x[0].reshape(3, 224, 224) for x in data]).astype('float32')
+            if len(np.array([x[1]
+                             for x in data]).astype('int64')) != batch_size:
+                continue
+            y_data = np.array([x[1] for x in data]).astype('int64').reshape(-1,
+                                                                            1)
+
+            avg_loss, acc_top1, acc_top5 = exe.run(
+                main_prog,
+                feed={"img": dy_x_data,
+                      "label": y_data},
+                fetch_list=[avg_loss_, acc_top1_, acc_top5_])
+
+            total_loss += avg_loss
+            total_acc1 += acc_top1
+            total_acc5 += acc_top5
+            total_sample += 1
+
+            end_time = time.time()
+            if batch_id % 2 == 0:
+                print( "epoch %d | batch step %d, loss %0.3f, acc1 %0.3f, acc5 %0.3f, time %f" % \
+                       ( epoch, batch_id, total_loss / total_sample, \
+                         total_acc1 / total_sample, total_acc5 / total_sample, end_time-start_time))
+            if batch_id == 10:
+                break
+
+
+def train_resnet_in_dygraph_mode():
```

Comment:
This function will not be executed that will reduce the code coverage. Please remove the function and add it in next PR.

Reply:
Thanks. Done.